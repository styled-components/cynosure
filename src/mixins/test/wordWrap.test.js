// @flow
import wordWrap from '../wordWrap'

describe('wordWrap', () => {
  it('should accept other values', () => {
    expect({
      ...wordWrap('break-all'),
    }).toEqual({
      overflowWrap: 'break-all',
      wordBreak: 'break-all',
      wordWrap: 'break-all',
    })
  })

  it('should default wrap to break-word', () => {
    expect({
      ...wordWrap(),
    }).toEqual({
      overflowWrap: 'break-word',
      wordBreak: 'break-all',
      wordWrap: 'break-word',
    })
  })
})
