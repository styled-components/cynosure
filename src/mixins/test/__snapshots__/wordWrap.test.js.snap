exports[`wordWrap should accept other values 1`] = `
Object {
  "overflowWrap": "break-all",
  "wordBreak": "break-all",
  "wordWrap": "break-all",
}
`;

exports[`wordWrap should default wrap to break-word 1`] = `
Object {
  "overflowWrap": "break-word",
  "wordBreak": "break-all",
  "wordWrap": "break-word",
}
`;
